src/components/components/FeaturedJobCard/FeaturedJobCard.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "./FeaturedJobCard.scss";
+
+function FeaturedJobCard({ job, logoNameToPath }) {
```

Comment:
You won't need LogoNameToPath here. Just access the logo like `<img src={job.logo}>`, the parent component will handle the rest :)

Reply:
done! thnx)